src/state.rs

```diff
@@ -346,21 +347,20 @@ pub(crate) enum TabExpandedString {
     NoTabs(Cow<'static, str>),
     WithTabs {
         original: Cow<'static, str>,
-        expanded: String,
+        expanded: OnceCell<String>,
```

Comment:
Can we just use `Option` instead of `OnceCell` for this? Seems simpler.

Reply:
No, I don't think so: We don't have `mut` access in `expanded`. AFAIK, that's essentially [what `OnceCell` is there for](https://old.reddit.com/r/rust/comments/126qaai/after_years_of_work_and_discussion_once_cell_has/jebnzf9/).